websites/B/bilibili/presence.ts

```diff
@@ -3,14 +3,6 @@
  * @param {Number} videoTime Current video time seconds
  * @param {Number} videoDuration Video duration seconds
  */
```

Comment:
I meant to delete these comments...

Reply:
Sorry for my poor English reading comprehension, I had just committed this, thanks